aws/eks/variables.tf

```diff
@@ -153,6 +153,12 @@ variable "external_secrets_settings" {
   default     = {}
 }
 
+variable "external_secrets_prefixes" {
+  description = "Set kubernetes_external_secrets role permissions to access AWS Secrets prefixes"
+  type        = list(string)
```

Comment:
As an alternative to moving this to its own parameter, you could also change the type of `external_secrets_settings` to `map(any)`, so it accepts lists as values as well. This will result in somewhat less type checking, but I don't think it would be a problem in practice.

Reply:
so I tried that yesterday, and it breaks because you have to pass a map of strings to the helm chart, and this parameter is used for the helm chart Values plus our one addition.

Reply:
so when passing `external_secrets_settings` to the helm chart, we'd need to drop that one array value (and assume the rest are strings) or force that one array value into a string. After working on both of those options for a bit (got further with the forcing the value into a string option), it just got ugly, so I backed up and decided to just keep the new variables since it is our additions & not helm chart variables, and that leaves the helm chart variables as is.

Reply:
see https://github.com/mozilla-it/terraform-modules/pull/154

Reply:
gonna reopen #154 with different branch & possibly open pr with branch that has the drop key option (which I couldn't get to work), then @smarnach & @duallain & @bkochendorfer can tell me y'alls preferences in the keep this change backwards compatible realm.